specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-11-20-preview/hanaonazure.json

```diff
@@ -0,0 +1,894 @@
+            "schema": {
+              "$ref": "#/definitions/SapMonitor"
+            }
+          },
+          "201": {
+            "description": "Creating SapMonitor",
+            "schema": {
+              "$ref": "#/definitions/SapMonitor"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "SapMonitors_Delete",
+        "x-ms-long-running-operation": true,
+        "summary": "Deletes a SAP monitor.",
+        "description": "Deletes a SAP monitor with the specified subscription, resource group, and monitor name.",
+        "x-ms-examples": {
+          "Deletes a SAP monitor": {
+            "$ref": "./examples/SapMonitors_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SapMonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted SapMonitor"
+          },
+          "202": {
+            "description": "Deleting SapMonitor"
+          },
+          "204": {
+            "description": "No SapMonitor to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "SapMonitors_Update",
+        "summary": "Patches the Tags or identity field of a SAP monitor.",
+        "description": "Patches the Tags or identity field of a SAP monitor for the specified subscription, resource group, and monitor name.",
+        "x-ms-examples": {
+          "Update Tags or Identity field of a SAP monitor": {
+            "$ref": "./examples/SapMonitors_Patch.json"
+          },
+          "Delete Tags field of a SAP monitor": {
+            "$ref": "./examples/SapMonitors_PatchTags_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SapMonitorNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsParameter"
```

Comment:
Your patch description says 'Patches the Tags or identity field of a SAP monitor.' But here you are referring to TagsParameter. Shouldn't it be TagsIdentityParameter here?